src/Components/DataCollection/supplierManagement.jsx

```diff
@@ -1,228 +1,89 @@
-// importing react features to use states, effects, and links
 import React, { useState, useEffect } from 'react';
-import { Link } from 'react-router-dom';
-// import database
 import { supabase } from '../../supabaseClient';
+import  Button from '../Common/CommonComponents/Button'
+import  Table from '../Common/CommonComponents/Table'
+import PopUp from '../Common/CommonComponents/PopUp'
 
-// declare + export supplierManagement component
 export default function SupplierManagement() {
-  // tableData - state var that stores all table data, initially an empty array
-  // setTableData - function used to update value of tableData
   const [tableData, setTableData] = useState([]);
-  // isPopupOpen - state var that changes depending on whether popup is open or not, initially set to false
-  // setIsPopupOpen - function used to update the value of isPopupOpen
   const [isPopupOpen, setIsPopupOpen] = useState(false);
-  // newRowData - state var that stores obj containing properties for new row of data, all props initially set to empty string
-  // setNewRowData - function to set new row data
   const [newRowData, setNewRowData] = useState({ supplier_name: '', location: '', key_product: '',  sustainability_score: '', key_contact: '', key_email: '' });
 
-  // useEffect - effect that runs once when component mounts
+  const fields = [
+    { link: true, id: 'supplier_name', label: 'Supplier Name', type: 'text' },
```

Comment:
lets move `link` to the end of this array like this 
{id: 'supplier_name', label: 'Supplier Name', type: 'text',  link: true  },